src/pijama_ast/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "pijama_ast"
+version = "0.1.0"
+authors = ["Christian Poveda <git@christianpoveda.xyz>", "Sean Chen <seanchen11235@gmail.com>"]
```

Comment:
I think we can change this to `authors = ["The Pijama Project Developers"]` instead, same for the root `Cargo.toml`